# libpq.po
#   Japanese message translation file for libpq
#
# Copyright (C) 2011-2024 PostgreSQL Global Development Group
#
# This file is distributed under the same license as the PostgreSQL package.
#
msgid ""
msgstr ""
"Project-Id-Version: libpq (PostgreSQL 18)\n"
"Report-Msgid-Bugs-To: pgsql-bugs@lists.postgresql.org\n"
"POT-Creation-Date: 2025-06-27 18:25+0900\n"
"PO-Revision-Date: 2025-06-30 13:15+0900\n"
"Last-Translator: Kyotaro Horiguchi <horikyota.ntt@gmail.com>\n"
"Language-Team: Japan PostgreSQL Users Group <jpug-doc@ml.postgresql.jp>\n"
"Language: ja\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=1; plural=0;\n"
"X-Generator: Poedit 1.8.13\n"

#: ../libpq-oauth/oauth-curl.c:306 ../libpq-oauth/oauth-curl.c:1936
#, c-format
msgid "libcurl easy handle removal failed: %s"
msgstr "libcurl簡易ハンドルの削除に失敗しました: %s"

#: ../libpq-oauth/oauth-curl.c:326
#, c-format
msgid "libcurl multi handle cleanup failed: %s"
msgstr "libcurlマルチハンドルのクリーンアップに失敗しました: %s"

#: ../libpq-oauth/oauth-curl.c:389 ../libpq-oauth/oauth-curl.c:400
#, c-format
msgid "failed to set %s on OAuth connection: %s"
msgstr "OAuth接続での%sの設定に失敗しました: %s"

#: ../libpq-oauth/oauth-curl.c:411
#, c-format
msgid "failed to get %s from OAuth response: %s"
msgstr "OAuth応答からの%sの取得に失敗しました: %s"

#: ../libpq-oauth/oauth-curl.c:514 ../libpq-oauth/oauth-curl.c:624
#, c-format
msgid "JSON is too deeply nested"
msgstr "JSONのネストが深すぎます"

#: ../libpq-oauth/oauth-curl.c:539
#, c-format
msgid "internal error: started field '%s' before field '%s' was finished"
msgstr "内部エラー: フィールド '%2$s' の終了前にフィールド '%1$s' が開始されました"

#: ../libpq-oauth/oauth-curl.c:566
#, c-format
msgid "field \"%s\" is duplicated"
msgstr "フィールド\"%s\"は重複しています。"

#: ../libpq-oauth/oauth-curl.c:591
#, c-format
msgid "internal error: field '%s' still active at end of object"
msgstr "内部エラー: オブジェクト終了時点でフィールド '%s' が未終了です"

#: ../libpq-oauth/oauth-curl.c:606 ../libpq-oauth/oauth-curl.c:666
#, c-format
msgid "top-level element must be an object"
msgstr "最上位要素はオブジェクトでなければなりません"

#: ../libpq-oauth/oauth-curl.c:647
#, c-format
msgid "internal error: found unexpected array end while parsing field '%s'"
msgstr "内部エラー: フィールド'%s'のパース中に予期しない配列がありました"

#: ../libpq-oauth/oauth-curl.c:702
#, c-format
msgid "internal error: scalar target found at nesting level %d"
msgstr "内部エラー: ネストレベル%dにーターゲットがありました"

#: ../libpq-oauth/oauth-curl.c:712
#, c-format
msgid "internal error: scalar field '%s' would be assigned twice"
msgstr "内部エラー: フィールド '%s' に2回代入されます"

#: ../libpq-oauth/oauth-curl.c:734
#, c-format
msgid "internal error: array member found at nesting level %d"
msgstr "内部エラー: ネストレベル%dに配列の要素がありました"

#: ../libpq-oauth/oauth-curl.c:769
#, c-format
msgid "no content type was provided"
msgstr "Content-Typeヘッダーがありませんでした"

#: ../libpq-oauth/oauth-curl.c:808
#, c-format
msgid "unexpected content type: \"%s\""
msgstr "想定しないContent-Typeヘッダーの値: \"%s\""

#: ../libpq-oauth/oauth-curl.c:833
#, c-format
msgid "response contains embedded NULLs"
msgstr "応答の途中にnullが含まれています"

#: ../libpq-oauth/oauth-curl.c:843
#, c-format
msgid "response is not valid UTF-8"
msgstr "応答は有効なUTF-8ではありません"

#: ../libpq-oauth/oauth-curl.c:883
#, c-format
msgid "field \"%s\" is missing"
msgstr "フィールド\"%s\"が存在しません"

#: ../libpq-oauth/oauth-curl.c:1117
#, c-format
msgid "provider rejected the oauth_client_secret"
msgstr "プロバイダがoauth_client_secretを拒否しました"

#: ../libpq-oauth/oauth-curl.c:1181
#, c-format
msgid "failed to create epoll set: %m"
msgstr "epollセットの作成に失敗しました: %m"

#: ../libpq-oauth/oauth-curl.c:1188
#, c-format
msgid "failed to create timerfd: %m"
msgstr "timerfd の作成に失敗しました: %m"

#: ../libpq-oauth/oauth-curl.c:1194
#, c-format
msgid "failed to add timerfd to epoll set: %m"
msgstr "epollセットへのtimerfdの追加に失敗しました: %m"

#. translator: the term "kqueue" (kernel queue) should not be translated
#: ../libpq-oauth/oauth-curl.c:1204
#, c-format
msgid "failed to create kqueue: %m"
msgstr "kqueueの作成に失敗しました: %m"

#: ../libpq-oauth/oauth-curl.c:1217
#, c-format
msgid "failed to create timer kqueue: %m"
msgstr "タイマーkqueueの作成に失敗しました: %m"

#: ../libpq-oauth/oauth-curl.c:1261 ../libpq-oauth/oauth-curl.c:1333
#, c-format
msgid "unknown libcurl socket operation: %d"
msgstr "未知のlibcurlソケット操作: %d"

#: ../libpq-oauth/oauth-curl.c:1278
#, c-format
msgid "could not add to epoll set: %m"
msgstr "epollセットに追加できませんでした: %m"

#: ../libpq-oauth/oauth-curl.c:1282
#, c-format
msgid "could not delete from epoll set: %m"
msgstr "epollセットから削除できませんでした: %m"

#: ../libpq-oauth/oauth-curl.c:1286
#, c-format
msgid "could not update epoll set: %m"
msgstr "epollセットの設定変更に失敗しました: %m"

#: ../libpq-oauth/oauth-curl.c:1340
#, c-format
msgid "could not modify kqueue: %m"
msgstr "kqueueの設定変更に失敗しました: %m"

#: ../libpq-oauth/oauth-curl.c:1364
#, c-format
msgid "could not delete from kqueue: %m"
msgstr "kqueueからの削除に失敗しました: %m"

#: ../libpq-oauth/oauth-curl.c:1367
#, c-format
msgid "could not add to kqueue: %m"
msgstr "kqueueへの追加に失敗しました: %m"

#: ../libpq-oauth/oauth-curl.c:1419
#, c-format
msgid "setting timerfd to %ld: %m"
msgstr "timerfdの%ldへの設定中にエラー: %m"

#: ../libpq-oauth/oauth-curl.c:1449
#, c-format
msgid "deleting kqueue timer: %m"
msgstr "kqueue タイマー削除中にエラー: %m"

#: ../libpq-oauth/oauth-curl.c:1456
#, c-format
msgid "removing kqueue timer from multiplexer: %m"
msgstr "マルチプレクサからのkqueueタイマー削除中にエラー: %m"

#: ../libpq-oauth/oauth-curl.c:1467
#, c-format
msgid "setting kqueue timer to %ld: %m"
msgstr "kqueueタイマーの%ldへの設定中にエラー: %m"

#: ../libpq-oauth/oauth-curl.c:1474
#, c-format
msgid "adding kqueue timer to multiplexer: %m"
msgstr "マルチプレクサへのkqueueタイマー追加中にエラー: %m"

#: ../libpq-oauth/oauth-curl.c:1497
#, c-format
msgid "getting timerfd value: %m"
msgstr "timerfdの値の取得中にエラー: %m"

#: ../libpq-oauth/oauth-curl.c:1519
#, c-format
msgid "checking kqueue for timeout: %m"
msgstr "kqueueのタイムアウトチェック中にエラー: %m"

#: ../libpq-oauth/oauth-curl.c:1651
#, c-format
msgid "failed to create libcurl multi handle"
msgstr "libcurl のマルチハンドルの作成に失敗しました"

#: ../libpq-oauth/oauth-curl.c:1671
#, c-format
msgid "failed to create libcurl handle"
msgstr "libcurl ハンドルの作成に失敗しました"

#: ../libpq-oauth/oauth-curl.c:1755 ../libpq-oauth/oauth-curl.c:1796
#: ../libpq-oauth/oauth-curl.c:2109 ../libpq-oauth/oauth-curl.c:2270
#: ../libpq-oauth/oauth-curl.c:2331 ../libpq-oauth/oauth-curl.c:2420
#: ../libpq-oauth/oauth-curl.c:2714 ../libpq-oauth/oauth-curl.c:2905
#: fe-auth-scram.c:374 fe-auth-scram.c:447 fe-auth-scram.c:599
#: fe-auth-scram.c:619 fe-auth-scram.c:643 fe-auth-scram.c:657
#: fe-auth-scram.c:703 fe-auth-scram.c:739 fe-auth-scram.c:931 fe-auth.c:308
#: fe-auth.c:382 fe-auth.c:416 fe-auth.c:694 fe-auth.c:827 fe-auth.c:1330
#: fe-auth.c:1493 fe-cancel.c:177 fe-connect.c:1011 fe-connect.c:1051
#: fe-connect.c:2171 fe-connect.c:2333 fe-connect.c:3726 fe-connect.c:5182
#: fe-connect.c:5495 fe-connect.c:5750 fe-connect.c:5868 fe-connect.c:6115
#: fe-connect.c:6195 fe-connect.c:6293 fe-connect.c:6544 fe-connect.c:6571
#: fe-connect.c:6647 fe-connect.c:6670 fe-connect.c:6694 fe-connect.c:6729
#: fe-connect.c:6815 fe-connect.c:6823 fe-connect.c:7180 fe-connect.c:7362
#: fe-exec.c:530 fe-exec.c:1315 fe-exec.c:3254 fe-exec.c:4293 fe-exec.c:4459
#: fe-gssapi-common.c:109 fe-lobj.c:870 fe-protocol3.c:210 fe-protocol3.c:233
#: fe-protocol3.c:256 fe-protocol3.c:273 fe-protocol3.c:294 fe-protocol3.c:368
#: fe-protocol3.c:732 fe-protocol3.c:972 fe-protocol3.c:1553
#: fe-protocol3.c:1862 fe-protocol3.c:2263 fe-secure-common.c:110
#: fe-secure-gssapi.c:506 fe-secure-gssapi.c:696 fe-secure-openssl.c:405
#: fe-secure-openssl.c:1131
#, c-format
msgid "out of memory"
msgstr "メモリ不足です"

#: ../libpq-oauth/oauth-curl.c:1783
#, c-format
msgid "response is too large"
msgstr "応答が大きすぎます"

#: ../libpq-oauth/oauth-curl.c:1825
#, c-format
msgid "failed to queue HTTP request: %s"
msgstr "HTTPリクエストのキューイングに失敗: %s"

#: ../libpq-oauth/oauth-curl.c:1842 ../libpq-oauth/oauth-curl.c:1895
#, c-format
msgid "asynchronous HTTP request failed: %s"
msgstr "非同期HTTPリクエストが失敗しました: %s"

#: ../libpq-oauth/oauth-curl.c:1947
#, c-format
msgid "no result was retrieved for the finished handle"
msgstr "終了済みのハンドルから結果が得られませんでした"

#: ../libpq-oauth/oauth-curl.c:2080 ../libpq-oauth/oauth-curl.c:2386
#: ../libpq-oauth/oauth-curl.c:2465
#, c-format
msgid "unexpected response code %ld"
msgstr "想定外のレスポンスコード %ld"

#: ../libpq-oauth/oauth-curl.c:2152
#, c-format
msgid "the issuer identifier (%s) does not match oauth_issuer (%s)"
msgstr "issuer識別子(%s)がoauth_issuer (%s)と一致しません"

#: ../libpq-oauth/oauth-curl.c:2179
#, c-format
msgid "issuer \"%s\" does not provide a device authorization endpoint"
msgstr "issuer\"%s\"はデバイス認証エンドポイントを提供していません"

#: ../libpq-oauth/oauth-curl.c:2205
#, c-format
msgid "device authorization endpoint \"%s\" must use HTTPS"
msgstr "デバイス認証エンドポイント\"%s\"はHTTPSを使用しなければなりません"

#: ../libpq-oauth/oauth-curl.c:2214
#, c-format
msgid "token endpoint \"%s\" must use HTTPS"
msgstr "トークン・エンドポイント \"%s\" はHTTPSを使用しなければなりません"

#: ../libpq-oauth/oauth-curl.c:2523
#, c-format
msgid "slow_down interval overflow"
msgstr "slow_down間隔のオーバーフロー"

#. translator: The first %s is a URL for the user to visit in a
#. browser, and the second %s is a code to be copy-pasted there.
#.
#: ../libpq-oauth/oauth-curl.c:2559
#, c-format
msgid "Visit %s and enter the code: %s\n"
msgstr "%sにアクセスしてコードを入力してください: %s\n"

#: ../libpq-oauth/oauth-curl.c:2564
#, c-format
msgid "device prompt failed"
msgstr "デバイス認証の開始に失敗しました"

#: ../libpq-oauth/oauth-curl.c:2620
#, c-format
msgid "curl_global_init previously failed during OAuth setup"
msgstr "curl_global_init は過去の OAuth セットアップ中に失敗しています"

#: ../libpq-oauth/oauth-curl.c:2639
#, c-format
msgid "curl_global_init failed during OAuth setup"
msgstr "curl_global_init が OAuth セットアップ中に失敗しました"

#: ../libpq-oauth/oauth-curl.c:2660
#, c-format
msgid ""
"libcurl is no longer thread-safe\n"
"\tCurl initialization was reported thread-safe when libpq\n"
"\twas compiled, but the currently installed version of\n"
"\tlibcurl reports that it is not. Recompile libpq against\n"
"\tthe installed version of libcurl."
msgstr ""
"libcurlはすでにスレッドセーフではありません\n"
"libpqのコンパイルの際にはcurlの初期化がスレッドセーフだと報告されましたが、\n"
"現在インストールされている libcurl はそうではないと報告しています。 libpq を\n"
"このインストールされているバージョンの libcurl でコンパイルしてください。"

#: fe-auth-scram.c:228
#, c-format
msgid "malformed SCRAM message (empty message)"
msgstr "SCRAMメッセージのフォーマット異常 (空のメッセージ)"

#: fe-auth-scram.c:233
#, c-format
msgid "malformed SCRAM message (length mismatch)"
msgstr "SCRAMメッセージのフォーマット異常 (長さの不整合)"

#: fe-auth-scram.c:277
#, c-format
msgid "could not verify server signature: %s"
msgstr "サーバー署名を検証できませんでした: %s"

#: fe-auth-scram.c:283
#, c-format
msgid "incorrect server signature"
msgstr "正しくないサーバー署名"

#: fe-auth-scram.c:292
#, c-format
msgid "invalid SCRAM exchange state"
msgstr "不正なSCRAM交換状態"

#: fe-auth-scram.c:316
#, c-format
msgid "malformed SCRAM message (attribute \"%c\" expected)"
msgstr "SCRAMメッセージのフォーマット異常 (属性 \"%c\" が必要)"

#: fe-auth-scram.c:325
#, c-format
msgid "malformed SCRAM message (expected character \"=\" for attribute \"%c\")"
msgstr "SCRAMメッセージのフォーマット異常 (属性 \"%c\" に文字 \"=\" が必要)"

#: fe-auth-scram.c:365
#, c-format
msgid "could not generate nonce"
msgstr "nonce を生成できませんでした"

#: fe-auth-scram.c:381
#, c-format
msgid "could not encode nonce"
msgstr "nonceをエンコードできませんでした"

#: fe-auth-scram.c:569
#, c-format
msgid "could not calculate client proof: %s"
msgstr "クライアント証明の算出に失敗しました: %s"

#: fe-auth-scram.c:584
#, c-format
msgid "could not encode client proof"
msgstr "クライアントの証明のエンコードに失敗しました"

#: fe-auth-scram.c:636
#, c-format
msgid "invalid SCRAM response (nonce mismatch)"
msgstr "不正なSCRAM応答 (nonce の不一致)"

#: fe-auth-scram.c:666
#, c-format
msgid "malformed SCRAM message (invalid salt)"
msgstr "SCRAMメッセージのフォーマット異常 (不正なソルト)"

#: fe-auth-scram.c:679
#, c-format
msgid "malformed SCRAM message (invalid iteration count)"
msgstr "SCRAMメッセージのフォーマット異常 (不正な繰り返し回数)"

#: fe-auth-scram.c:684
#, c-format
msgid "malformed SCRAM message (garbage at end of server-first-message)"
msgstr "SCRAMメッセージのフォーマット異常 (server-first-message 終端の余分なデータ)"

#: fe-auth-scram.c:718
#, c-format
msgid "error received from server in SCRAM exchange: %s"
msgstr "SCRAM交換中にサーバーからのエラーを受信しました: %s"

#: fe-auth-scram.c:733
#, c-format
msgid "malformed SCRAM message (garbage at end of server-final-message)"
msgstr "SCRAMメッセージのフォーマット異常 (server-final-message 終端の余分なデータ)"

#: fe-auth-scram.c:750
#, c-format
msgid "malformed SCRAM message (invalid server signature)"
msgstr "SCRAMメッセージのフォーマット異常 (不正なサーバー署名)"

#: fe-auth-scram.c:940
msgid "could not generate random salt"
msgstr "乱数ソルトを生成できませんでした"

#: fe-auth.c:80
#, c-format
msgid "out of memory allocating GSSAPI buffer (%d)"
msgstr "GSSAPIバッファの割り当ての際のメモリ不足(%d)"

#: fe-auth.c:146
msgid "GSSAPI continuation error"
msgstr "GSSAI続行エラー"

#: fe-auth.c:176 fe-auth.c:410 fe-gssapi-common.c:97 fe-secure-common.c:99
#: fe-secure-common.c:173
#, c-format
msgid "host name must be specified"
msgstr "ホスト名を指定しなければなりません"

#: fe-auth.c:182
#, c-format
msgid "duplicate GSS authentication request"
msgstr "重複するGSS認証要求"

#: fe-auth.c:246
#, c-format
msgid "out of memory allocating SSPI buffer (%d)"
msgstr "SSPIバッファの割り当ての際のメモリ不足(%d)"

#: fe-auth.c:297
msgid "SSPI continuation error"
msgstr "SSPI続行エラー"

#: fe-auth.c:372
#, c-format
msgid "duplicate SSPI authentication request"
msgstr "重複したSSPI認証要求"

#: fe-auth.c:397
msgid "could not acquire SSPI credentials"
msgstr "SSPI資格を入手できませんでした"

#: fe-auth.c:449
#, c-format
msgid "channel binding required, but SSL not in use"
msgstr "チャネルバインディングが要求されていますが、SSLが使用されていません"

#: fe-auth.c:455
#, c-format
msgid "duplicate SASL authentication request"
msgstr "重複するSASL認証要求"

#: fe-auth.c:513
#, c-format
msgid "channel binding is required, but client does not support it"
msgstr "チャネルバインディングが要求されていますが、クライアントがサポートしていません"

#: fe-auth.c:529
#, c-format
msgid "server offered SCRAM-SHA-256-PLUS authentication over a non-SSL connection"
msgstr "サーバーが非SSL接続上で SCRAM-SHA-256-PLUS 認証を提示してきました"

#: fe-auth.c:551
#, c-format
msgid "none of the server's SASL authentication mechanisms are supported"
msgstr "サーバー側のいずれのSASL認証機構もサポートされていません"

#: fe-auth.c:571
#, c-format
msgid "authentication method requirement \"%s\" failed: server requested %s authentication"
msgstr "必須の認証方式\"%s\"を満たせませんでした: サーバーは %s 認証を要求しました"

#: fe-auth.c:580
#, c-format
msgid "channel binding is required, but server did not offer an authentication method that supports channel binding"
msgstr "チャネルバインディングが要求されていますが、サーバーがチャネルバインディングをサポートする認証方式を提供しませんでした"

#: fe-auth.c:716
#, c-format
msgid "out of memory allocating SASL buffer (%d)"
msgstr "SASLバッファの割り当ての際のメモリ不足(%d)"

#: fe-auth.c:758
#, c-format
msgid "AuthenticationSASLFinal received from server, but SASL authentication was not completed"
msgstr "サーバーからAuthenticationSASLFinalを受信しました、しかしSASL認証は完了していません"

#: fe-auth.c:768
#, c-format
msgid "no client response found after SASL exchange success"
msgstr "SASL交換の成功後にクライアントからの応答がありません"

#: fe-auth.c:836 fe-auth.c:843 fe-auth.c:1476 fe-auth.c:1487
#, c-format
msgid "could not encrypt password: %s"
msgstr "パスワードを暗号化できませんでした: %s"

#: fe-auth.c:873
msgid "server requested a cleartext password"
msgstr "サーバーが平文パスワードを要求してきました"

#: fe-auth.c:875
msgid "server requested a hashed password"
msgstr "サーバーがハッシュ化パスワードを要求してきました"

#: fe-auth.c:878
msgid "server requested GSSAPI authentication"
msgstr "サーバーがGSSAPI認証を要求してきました"

#: fe-auth.c:880
msgid "server requested SSPI authentication"
msgstr "サーバーがSSPI認証を要求してきました"

#: fe-auth.c:884
msgid "server requested SASL authentication"
msgstr "サーバーがSASL認証を要求してきました"

#: fe-auth.c:887
msgid "server requested an unknown authentication type"
msgstr "サーバーが不明な認証タイプを要求してきました"

#: fe-auth.c:920
#, c-format
msgid "server did not request an SSL certificate"
msgstr "サーバーがSSL証明書を要求してきませんでした"

#: fe-auth.c:925
#, c-format
msgid "server accepted connection without a valid SSL certificate"
msgstr "サーバーは有効なSSL証明書なしで接続を受け付けました"

#: fe-auth.c:979
msgid "server did not complete authentication"
msgstr "サーバーが認証を完了しませんでした"

#: fe-auth.c:1013
#, c-format
msgid "authentication method requirement \"%s\" failed: %s"
msgstr "必須の認証方式\"%s\"が失敗しました: %s"

#: fe-auth.c:1036
#, c-format
msgid "channel binding required, but server authenticated client without channel binding"
msgstr "チャネルバインディングが要求されていますが、サーバーはチャネルバインディングを使用せずに認証を行いました"

#: fe-auth.c:1041
#, c-format
msgid "channel binding required but not supported by server's authentication request"
msgstr "チャネルバインディングが要求されていますが、サーバーの認証要求ではサポートされていません"

#: fe-auth.c:1081
#, c-format
msgid "Kerberos 4 authentication not supported"
msgstr "Kerberos 4認証はサポートされていません"

#: fe-auth.c:1085
#, c-format
msgid "Kerberos 5 authentication not supported"
msgstr "Kerberos 5認証はサポートされていません"

#: fe-auth.c:1155
#, c-format
msgid "GSSAPI authentication not supported"
msgstr "GSSAPI認証はサポートされていません"

#: fe-auth.c:1186
#, c-format
msgid "SSPI authentication not supported"
msgstr "SSPI認証はサポートされていません"

#: fe-auth.c:1193
#, c-format
msgid "Crypt authentication not supported"
msgstr "Crypt認証はサポートされていません"

#: fe-auth.c:1267
#, c-format
msgid "authentication method %u not supported"
msgstr "認証方式%uはサポートされていません"

#: fe-auth.c:1307
#, c-format
msgid "user name lookup failure: error code %lu"
msgstr "ユーザー名の参照に失敗: エラーコード %lu"

#: fe-auth.c:1315
#, c-format
msgid "could not look up local user ID %ld: %m"
msgstr "ローカルユーザーID %ldの参照に失敗しました: %m"

#: fe-auth.c:1320
#, c-format
msgid "local user with ID %ld does not exist"
msgstr "ID %ld を持つローカルユーザーは存在しません"

#: fe-auth.c:1439
#, c-format
msgid "unexpected shape of result set returned for SHOW"
msgstr "SHOW に対する予期しない形のリザルトセット"

#: fe-auth.c:1447
#, c-format
msgid "\"password_encryption\" value too long"
msgstr "\"password_encryption\"の値が長すぎます"

#: fe-auth.c:1497
#, c-format
msgid "unrecognized password encryption algorithm \"%s\""
msgstr "認識できないパスワード暗号化アルゴリズム \"%s\""

#: fe-cancel.c:79
#, c-format
msgid "connection pointer is NULL"
msgstr "接続ポインタはNULLです"

#: fe-cancel.c:85 fe-misc.c:613
#, c-format
msgid "connection not open"
msgstr "接続はオープンされていません"

#: fe-cancel.c:92
#, c-format
msgid "no cancellation key received"
msgstr "キャンセルキーを受信していません"

#: fe-cancel.c:211
#, c-format
msgid "cancel request is already being sent on this connection"
msgstr "このセッションではすでにキャンセル要求を送出済みです"

#: fe-cancel.c:281
#, c-format
msgid "unexpected response from server"
msgstr "サーバーからの想定外の応答"

#: fe-connect.c:1308
#, c-format
msgid "could not match %d host names to %d hostaddr values"
msgstr "%d個のホスト名と%d個のhostaddrの値との突き合せはできません"

#: fe-connect.c:1388
#, c-format
msgid "could not match %d port numbers to %d hosts"
msgstr "%d個のポート番号と%d個のホストとの突き合せはできません"

#: fe-connect.c:1516
#, c-format
msgid "negative require_auth method \"%s\" cannot be mixed with non-negative methods"
msgstr "require_authの方式否定\"%s\"は方式要求と同時に指定することはできません"

#: fe-connect.c:1529
#, c-format
msgid "require_auth method \"%s\" cannot be mixed with negative methods"
msgstr "require_authの方式要求\"%s\"は方式否定と同時に指定することはできません"

#: fe-connect.c:1605 fe-connect.c:1734 fe-connect.c:1776 fe-connect.c:1819
#: fe-connect.c:1922 fe-connect.c:1968 fe-connect.c:2008 fe-connect.c:2075
#: fe-connect.c:8254
#, c-format
msgid "invalid %s value: \"%s\""
msgstr "%s の値が不正: \"%s\""

#: fe-connect.c:1647
#, c-format
msgid "internal error: no space in allowed_sasl_mechs"
msgstr "内部エラー: allowed_sasl_mechs に空きがありません"

#: fe-connect.c:1686
#, c-format
msgid "require_auth method \"%s\" is specified more than once"
msgstr "require_authで方式\"%s\"が複数回指定されました"

#: fe-connect.c:1757 fe-connect.c:1796 fe-connect.c:1828 fe-connect.c:1930
#, c-format
msgid "%s value \"%s\" invalid when SSL support is not compiled in"
msgstr "%s 値\"%s\"はSSLサポートがコンパイルされていない場合は無効です"

#: fe-connect.c:1848
#, c-format
msgid "weak sslmode \"%s\" may not be used with sslnegotiation=direct (use \"require\", \"verify-ca\", or \"verify-full\")"
msgstr "弱いsslmode\"%s\"はsslnegotiation=direct とともには使用できません( \"require\", \"verify-ca\"または\"verify-full\"を使用してください)"

#: fe-connect.c:1870
#, c-format
msgid "weak sslmode \"%s\" may not be used with sslrootcert=system (use \"verify-full\")"
msgstr "弱いsslmode\"%s\"はsslrootcert=systemとともには使用できません(\"verify-full\"を使用してください)"

#: fe-connect.c:1883 fe-connect.c:1891
#, c-format
msgid "invalid \"%s\" value: \"%s\""
msgstr "\"%s\"の値が不正: \"%s\""

#: fe-connect.c:1908
#, c-format
msgid "invalid SSL protocol version range"
msgstr "不正なSSLプロトコルバージョン範囲"

#: fe-connect.c:1945
#, c-format
msgid "%s value \"%s\" is not supported (check OpenSSL version)"
msgstr "%s 値\"%s\"はサポートされていません(OpenSSLのバージョンを確認してください)"

#: fe-connect.c:1975
#, c-format
msgid "gssencmode value \"%s\" invalid when GSSAPI support is not compiled in"
msgstr "gssencmodeの値\"%s\"はGSSAPIサポートがコンパイルされていない場合は不正"

#: fe-connect.c:2029
#, c-format
msgid "invalid SCRAM client key"
msgstr "不正なSCRAMクライアントキー"

#: fe-connect.c:2034
#, c-format
msgid "invalid SCRAM client key length: %d"
msgstr "不正なSCRAMクライアントキー長: %d"

#: fe-connect.c:2052
#, c-format
msgid "invalid SCRAM server key"
msgstr "不正なSCRAMサーバーキー"

#: fe-connect.c:2057
#, c-format
msgid "invalid SCRAM server key length: %d"
msgstr "不正なSCRAMサーバーキー長: %d"

#: fe-connect.c:2144
#, c-format
msgid "\"%s\" is greater than \"%s\""
msgstr "\"%s\"が\"%s\"より大きいです"

#: fe-connect.c:2356
#, c-format
msgid "could not set socket to TCP no delay mode: %s"
msgstr "TCPソケットを非遅延モードに設定できませんでした: %s"

#: fe-connect.c:2415
#, c-format
msgid "connection to server on socket \"%s\" failed: "
msgstr "ソケット\"%s\"のサーバーへの接続に失敗しました: "

#: fe-connect.c:2441
#, c-format
msgid "connection to server at \"%s\" (%s), port %s failed: "
msgstr "\"%s\"(%s)、ポート%sのサーバーへの接続に失敗しました: "

#: fe-connect.c:2446
#, c-format
msgid "connection to server at \"%s\", port %s failed: "
msgstr "\"%s\"、ポート%sのサーバーへの接続に失敗しました: "

#: fe-connect.c:2469
#, c-format
msgid "\tIs the server running locally and accepting connections on that socket?"
msgstr "\tサーバーはローカルで稼働していてそのソケットで接続を受け付けていますか?"

#: fe-connect.c:2471
#, c-format
msgid "\tIs the server running on that host and accepting TCP/IP connections?"
msgstr "\tサーバーはそのホスト上で稼働していてTCP/IP接続を受け付けていますか?"

#: fe-connect.c:2517 fe-connect.c:2551 fe-connect.c:2586 fe-connect.c:2684
#: fe-connect.c:3410
#, c-format
msgid "%s(%s) failed: %s"
msgstr "%s(%s)が失敗しました: %s"

#: fe-connect.c:2650
#, c-format
msgid "%s(%s) failed: error code %d"
msgstr "%s(%s)が失敗しました: エラーコード %d"

#: fe-connect.c:2962
#, c-format
msgid "invalid connection state, probably indicative of memory corruption"
msgstr "接続状態が不正です。メモリ障害の可能性があります"

#: fe-connect.c:3045
#, c-format
msgid "invalid port number: \"%s\""
msgstr "不正なポート番号です: \"%s\""

#: fe-connect.c:3059
#, c-format
msgid "could not translate host name \"%s\" to address: %s"
msgstr "ホスト名\"%s\"をアドレスに変換できませんでした: %s"

#: fe-connect.c:3071
#, c-format
msgid "could not parse network address \"%s\": %s"
msgstr "ネットワークアドレス\"%s\"をパースできませんでした: %s"

#: fe-connect.c:3082
#, c-format
msgid "Unix-domain socket path \"%s\" is too long (maximum %d bytes)"
msgstr "Unixドメインソケットのパス\"%s\"が長すぎます(最大 %d バイト)"

#: fe-connect.c:3096
#, c-format
msgid "could not translate Unix-domain socket path \"%s\" to address: %s"
msgstr "Unixドメインソケットのパス\"%s\"をアドレスに変換できませんでした: %s"

#: fe-connect.c:3262 fe-connect.c:4709
#, c-format
msgid "GSSAPI encryption required but it is not supported over a local socket"
msgstr "GSSAPI暗号化が要求されていますが、ローカルソケットではサポートされません"

#: fe-connect.c:3270 fe-connect.c:4838
#, c-format
msgid "GSSAPI encryption required but no credential cache"
msgstr "GSSAPI暗号化が要求されましたが、資格証明キャッシュがありません"

#: fe-connect.c:3338
#, c-format
msgid "could not create socket: %s"
msgstr "ソケットを作成できませんでした: %s"

#: fe-connect.c:3369
#, c-format
msgid "could not set socket to nonblocking mode: %s"
msgstr "ソケットを非ブロッキングモードに設定できませんでした: %s\\"

#: fe-connect.c:3380
#, c-format
msgid "could not set socket to close-on-exec mode: %s"
msgstr "ソケットをclose-on-execモードに設定できませんでした: %s"

#: fe-connect.c:3537
#, c-format
msgid "could not get socket error status: %s"
msgstr "ソケットのエラー状態を取得できませんでした: %s"

#: fe-connect.c:3564
#, c-format
msgid "could not get client address from socket: %s"
msgstr "ソケットからクライアントアドレスを取得できませんでした: %s"

#: fe-connect.c:3590
#, c-format
msgid "requirepeer parameter is not supported on this platform"
msgstr "このプラットフォームでは requirepeer パラメータはサポートされていません"

#: fe-connect.c:3592
#, c-format
msgid "could not get peer credentials: %s"
msgstr "接続先の資格証明を取得できませんでした: %s"

#: fe-connect.c:3605
#, c-format
msgid "requirepeer specifies \"%s\", but actual peer user name is \"%s\""
msgstr "requirepeerは\"%s\"を指定していますが、実際の接続先名は\"%s\"です"

#: fe-connect.c:3643
#, c-format
msgid "could not send GSSAPI negotiation packet: %s"
msgstr "GSSAPIネゴシエーションパケットを送信できませんでした: %s"

#: fe-connect.c:3682
#, c-format
msgid "could not send SSL negotiation packet: %s"
msgstr "SSLネゴシエーションパケットを送信できませんでした: %s"

#: fe-connect.c:3708
#, c-format
msgid "could not send cancel packet: %s"
msgstr "キャンセルパケットを送信できませんでした: %s"

#: fe-connect.c:3738
#, c-format
msgid "could not send startup packet: %s"
msgstr "開始パケットを送信できませんでした: %s"

#: fe-connect.c:3811
msgid "server does not support SSL, but SSL was required"
msgstr "サーバーはSSLをサポートしていませんが、要求されています"

#: fe-connect.c:3821
#, c-format
msgid "server sent an error response during SSL exchange"
msgstr "SSLハンドシェイク中にサーバーからエラー応答が返されました"

#: fe-connect.c:3826
#, c-format
msgid "received invalid response to SSL negotiation: %c"
msgstr "SSLネゴシエーションに対して不正な応答を受信しました: %c"

#: fe-connect.c:3846
#, c-format
msgid "received unencrypted data after SSL response"
msgstr "SSL応答の後に非暗号化データを受信しました"

#: fe-connect.c:3909
#, c-format
msgid "server sent an error response during GSS encryption exchange"
msgstr "GSS暗号化ハンドシェイク中にサーバーからエラー応答が返されました"

#: fe-connect.c:3927
msgid "server doesn't support GSSAPI encryption, but it was required"
msgstr "サーバーがGSSAPI暗号化サポートしていませんが、要求されてます"

#: fe-connect.c:3931
#, c-format
msgid "received invalid response to GSSAPI negotiation: %c"
msgstr "GSSAPIネゴシエーションに対して不正な応答を受信しました: %c"

#: fe-connect.c:3953
#, c-format
msgid "received unencrypted data after GSSAPI encryption response"
msgstr "GSSAPI暗号化応答の後に非暗号化データを受信しました"

#: fe-connect.c:4014
#, c-format
msgid "expected authentication request from server, but received %c"
msgstr "サーバーからの認証要求を想定していましたが、%cを受信しました"

#: fe-connect.c:4042 fe-connect.c:4174
#, c-format
msgid "received invalid authentication request"
msgstr "不正な認証要求を受信しました"

#: fe-connect.c:4048
#, c-format
msgid "received invalid protocol negotiation message"
msgstr "不正なプロトコルネゴシエーションメッセージを受信しました"

#: fe-connect.c:4067 fe-connect.c:4121
#, c-format
msgid "received invalid error message"
msgstr "不正なエラーメッセージを受信しました"

#: fe-connect.c:4151
#, c-format
msgid "received duplicate protocol negotiation message"
msgstr "重複したプロトコルネゴシエーションメッセージを受信しました"

#: fe-connect.c:4253
#, c-format
msgid "internal error: async authentication has no handler"
msgstr "内部エラー: 非同期認証のハンドラーがありません"

#: fe-connect.c:4278
#, c-format
msgid "internal error: async cleanup did not release polling socket"
msgstr "内部エラー: 非同期クリーンアップがポーリング中のソケットを解放しませんでした"

#: fe-connect.c:4301
#, c-format
msgid "internal error: async authentication did not set a socket for polling"
msgstr "内部エラー: 非同期認証がポーリング対象のソケットを設定しませんでした"

#: fe-connect.c:4334
#, c-format
msgid "unexpected message from server during startup"
msgstr "起動中にサーバーから想定外のメッセージがありました"

#: fe-connect.c:4425
#, c-format
msgid "session is read-only"
msgstr "セッションは読み取り専用です"

#: fe-connect.c:4427
#, c-format
msgid "session is not read-only"
msgstr "セッションは読み取り専用ではありません"

#: fe-connect.c:4480
#, c-format
msgid "server is in hot standby mode"
msgstr "サーバーはホットスタンバイモードです"

#: fe-connect.c:4482
#, c-format
msgid "server is not in hot standby mode"
msgstr "サーバーはスタンバイモードではありません"

#: fe-connect.c:4607 fe-connect.c:4657
#, c-format
msgid "\"%s\" failed"
msgstr "\"%s\"が失敗しました"

#: fe-connect.c:4671
#, c-format
msgid "invalid connection state %d, probably indicative of memory corruption"
msgstr "接続状態%dは不正です。メモリ障害の可能性があります"

#: fe-connect.c:5508
#, c-format
msgid "invalid LDAP URL \"%s\": scheme must be ldap://"
msgstr "不正なLDAP URL\"%s\":スキームはldap://でなければなりません"

#: fe-connect.c:5523
#, c-format
msgid "invalid LDAP URL \"%s\": missing distinguished name"
msgstr "不正なLDAP URL \"%s\": 識別名がありません"

#: fe-connect.c:5535 fe-connect.c:5593
#, c-format
msgid "invalid LDAP URL \"%s\": must have exactly one attribute"
msgstr "不正なLDAP URL \"%s\": ちょうど1つの属性を持たなければなりません"

#: fe-connect.c:5547 fe-connect.c:5609
#, c-format
msgid "invalid LDAP URL \"%s\": must have search scope (base/one/sub)"
msgstr "不正なLDAP URL \"%s\": 検索スコープ(base/one/sub)を持たなければなりません"

#: fe-connect.c:5559
#, c-format
msgid "invalid LDAP URL \"%s\": no filter"
msgstr "不正なLDAP URL \"%s\": フィルタがありません"

#: fe-connect.c:5581
#, c-format
msgid "invalid LDAP URL \"%s\": invalid port number"
msgstr "不正なLDAP URL \"%s\": ポート番号が不正です"

#: fe-connect.c:5618
#, c-format
msgid "could not create LDAP structure"
msgstr "LDAP構造体を作成できませんでした"

#: fe-connect.c:5693
#, c-format
msgid "lookup on LDAP server failed: %s"
msgstr "LDAPサーバーで検索に失敗しました: %s"

#: fe-connect.c:5703
#, c-format
msgid "more than one entry found on LDAP lookup"
msgstr "LDAP参照で複数のエントリが見つかりました"

#: fe-connect.c:5705 fe-connect.c:5716
#, c-format
msgid "no entry found on LDAP lookup"
msgstr "LDAP参照でエントリが見つかりません"

#: fe-connect.c:5726 fe-connect.c:5738
#, c-format
msgid "attribute has no values on LDAP lookup"
msgstr "LDAP参照で属性に値がありません"

#: fe-connect.c:5789 fe-connect.c:5808 fe-connect.c:6332
#, c-format
msgid "missing \"=\" after \"%s\" in connection info string"
msgstr "接続情報文字列において\"%s\"の後に\"=\"がありませんでした"

#: fe-connect.c:5879 fe-connect.c:6515 fe-connect.c:7345
#, c-format
msgid "invalid connection option \"%s\""
msgstr "不正な接続オプション\"%s\""

#: fe-connect.c:5894 fe-connect.c:6380
#, c-format
msgid "unterminated quoted string in connection info string"
msgstr "接続情報文字列内の閉じていない引用符"

#: fe-connect.c:5974
#, c-format
msgid "definition of service \"%s\" not found"
msgstr "サービス定義\"%s\"がみつかりません"

#: fe-connect.c:6000
#, c-format
msgid "service file \"%s\" not found"
msgstr "サービスファイル\"%s\"がみつかりません"

#: fe-connect.c:6013
#, c-format
msgid "line %d too long in service file \"%s\""
msgstr "サービスファイル\"%2$s\"の行%1$dが長すぎます"

#: fe-connect.c:6084 fe-connect.c:6127
#, c-format
msgid "syntax error in service file \"%s\", line %d"
msgstr "サービスファイル\"%s\"の行%dで構文エラー"

#: fe-connect.c:6095
#, c-format
msgid "nested service specifications not supported in service file \"%s\", line %d"
msgstr "サービスファイル\"%s\"、行%dでのネストしたサービス指定はサポートされていません"

#: fe-connect.c:6834
#, c-format
msgid "invalid URI propagated to internal parser routine: \"%s\""
msgstr "内部パーサ処理へ伝播した不正なURI: \"%s\""

#: fe-connect.c:6911
#, c-format
msgid "end of string reached when looking for matching \"]\" in IPv6 host address in URI: \"%s\""
msgstr "URI \"%s\"内のIPv6ホストアドレスにおいて対応する\"]\"を探している間に文字列が終わりました"

#: fe-connect.c:6918
#, c-format
msgid "IPv6 host address may not be empty in URI: \"%s\""
msgstr "URI内ではIPv6ホストアドレスは空であってはなりません: \"%s\""

#: fe-connect.c:6933
#, c-format
msgid "unexpected character \"%c\" at position %d in URI (expected \":\" or \"/\"): \"%s\""
msgstr "URI内の位置%2$dに想定外の文字\"%1$c\"があります(\":\"または\"/\"を期待していました): \"%3$s\""

#: fe-connect.c:7062
#, c-format
msgid "extra key/value separator \"=\" in URI query parameter: \"%s\""
msgstr "URI問い合わせパラメータ内にキーと値を分ける\"=\"が余分にあります: \"%s\""

#: fe-connect.c:7082
#, c-format
msgid "missing key/value separator \"=\" in URI query parameter: \"%s\""
msgstr "URI問い合わせパラメータ内にキーと値を分ける\\\"=\\\"がありません: \"%s\""

#: fe-connect.c:7134
#, c-format
msgid "invalid URI query parameter: \"%s\""
msgstr "不正なURI問い合わせパラメータ:\"%s\""

#: fe-connect.c:7218
#, c-format
msgid "invalid percent-encoded token: \"%s\""
msgstr "不正なパーセント符号化トークン: \"%s\""

#: fe-connect.c:7228
#, c-format
msgid "forbidden value %%00 in percent-encoded value: \"%s\""
msgstr "パーセント符号化された値では値%%00は許されません: \"%s\""

#: fe-connect.c:7250
#, c-format
msgid "unexpected spaces found in \"%s\", use percent-encoded spaces (%%20) instead"
msgstr "\"%s\"に予期しない空白文字があります、代わりにパーセントエンコードされた空白文字(%%20)を使用してください"

#: fe-connect.c:7632
msgid "connection pointer is NULL\n"
msgstr "接続ポインタはNULLです\n"

#: fe-connect.c:7640 fe-exec.c:713 fe-exec.c:975 fe-exec.c:3459
#: fe-protocol3.c:987 fe-protocol3.c:1020
msgid "out of memory\n"
msgstr "メモリ不足\n"

#: fe-connect.c:7942
#, c-format
msgid "WARNING: password file \"%s\" is not a plain file\n"
msgstr "WARNING: パスワードファイル\"%s\"がテキストファイルではありません\n"

#: fe-connect.c:7952
#, c-format
msgid "WARNING: password file \"%s\" has group or world access; permissions should be u=rw (0600) or less\n"
msgstr "警告: パスワードファイル \"%s\" がグループメンバもしくは他のユーザーから読める状態になっています。この権限はu=rw (0600)以下にすべきです\n"

#: fe-connect.c:8056
#, c-format
msgid "password retrieved from file \"%s\""
msgstr "パスワードはファイル\"%s\"から取り出しました"

#: fe-connect.c:8222
#, c-format
msgid "invalid integer value \"%s\" for connection option \"%s\""
msgstr "接続オプション\"%2$s\"に対する不正な整数値\"%1$s\""

#: fe-exec.c:469 fe-exec.c:3533
#, c-format
msgid "row number %d is out of range 0..%d"
msgstr "行番号%dは0..%dの範囲を超えています"

#: fe-exec.c:531 fe-protocol3.c:2068
#, c-format
msgid "%s"
msgstr "%s"

#: fe-exec.c:834
#, c-format
msgid "write to server failed"
msgstr "サーバーへの書き込みに失敗"

#: fe-exec.c:874
#, c-format
msgid "no error text available"
msgstr "エラー文字列がありません"

#: fe-exec.c:963
msgid "NOTICE"
msgstr "注意"

#: fe-exec.c:1021
msgid "PGresult cannot support more than INT_MAX tuples"
msgstr "PGresultはINT_MAX個以上のタプルを扱えません"

#: fe-exec.c:1033
msgid "size_t overflow"
msgstr "size_t オーバーフロー"

#: fe-exec.c:1438 fe-exec.c:1507 fe-exec.c:1553
#, c-format
msgid "command string is a null pointer"
msgstr "コマンド文字列がヌルポインタです"

#: fe-exec.c:1444 fe-exec.c:3003
#, c-format
msgid "%s not allowed in pipeline mode"
msgstr "%sはパイプラインモードでは使用できません"

#: fe-exec.c:1512 fe-exec.c:1558 fe-exec.c:1652
#, c-format
msgid "number of parameters must be between 0 and %d"
msgstr "パラメータ数は0から%dまでの間でなければなりません"

#: fe-exec.c:1548 fe-exec.c:1647
#, c-format
msgid "statement name is a null pointer"
msgstr "文の名前がヌルポインタです"

#: fe-exec.c:1689 fe-exec.c:3379
#, c-format
msgid "no connection to the server"
msgstr "サーバーへの接続がありません"

#: fe-exec.c:1697 fe-exec.c:3387
#, c-format
msgid "another command is already in progress"
msgstr "他のコマンドがすでに処理中です"

#: fe-exec.c:1727
#, c-format
msgid "cannot queue commands during COPY"
msgstr "COPY中はコマンドのキューイングはできません"

#: fe-exec.c:1846
#, c-format
msgid "length must be given for binary parameter"
msgstr "バイナリパラメータには長さを指定する必要があります"

#: fe-exec.c:2205
#, c-format
msgid "unexpected asyncStatus: %d"
msgstr "想定外のasyncStatus: %d"

#: fe-exec.c:2361
#, c-format
msgid "synchronous command execution functions are not allowed in pipeline mode"
msgstr "同期的にコマンドを実行する関数はパイプラインモード中は実行できません"

#: fe-exec.c:2378
msgid "COPY terminated by new PQexec"
msgstr "新たなPQexec\"によりCOPYが終了しました"

#: fe-exec.c:2394
#, c-format
msgid "PQexec not allowed during COPY BOTH"
msgstr "COPY BOTH 実行中の PQexec は許可されていません"

#: fe-exec.c:2630
#, c-format
msgid "unrecognized message type \"%c\""
msgstr "認識できないメッセージタイプ\"%c\""

#: fe-exec.c:2702 fe-exec.c:2756 fe-exec.c:2824 fe-protocol3.c:1999
#, c-format
msgid "no COPY in progress"
msgstr "実行中のCOPYはありません"

#: fe-exec.c:3010
#, c-format
msgid "connection in wrong state"
msgstr "接続状態が異常です"

#: fe-exec.c:3053
#, c-format
msgid "cannot enter pipeline mode, connection not idle"
msgstr "パイプラインモードに入れません、接続がアイドル状態ではありません"

#: fe-exec.c:3089 fe-exec.c:3110
#, c-format
msgid "cannot exit pipeline mode with uncollected results"
msgstr "未回収の結果が残っている状態でパイプラインモードを抜けることはできません"

#: fe-exec.c:3093
#, c-format
msgid "cannot exit pipeline mode while busy"
msgstr "ビジー状態でパイプラインモードを抜けることはできません"

#: fe-exec.c:3104
#, c-format
msgid "cannot exit pipeline mode while in COPY"
msgstr "COPY実行中にパイプラインモードを抜けることはできません"

#: fe-exec.c:3303
#, c-format
msgid "cannot send pipeline when not in pipeline mode"
msgstr "パイプラインモード外でパイプライン送出はできません"

#: fe-exec.c:3422
msgid "invalid ExecStatusType code"
msgstr "ExecStatusTypeコードが不正です"

#: fe-exec.c:3449
msgid "PGresult is not an error result\n"
msgstr "PGresutがエラー結果ではありません\n"

#: fe-exec.c:3517 fe-exec.c:3540
#, c-format
msgid "column number %d is out of range 0..%d"
msgstr "列番号%dは0..%dの範囲を超えています"

#: fe-exec.c:3555
#, c-format
msgid "parameter number %d is out of range 0..%d"
msgstr "パラメータ%dは0..%dの範囲を超えています"

#: fe-exec.c:3866
#, c-format
msgid "could not interpret result from server: %s"
msgstr "サーバーからの結果を解釈できませんでした: %s"

#: fe-exec.c:4141 fe-exec.c:4255
#, c-format
msgid "incomplete multibyte character"
msgstr "不完全なマルチバイト文字"

#: fe-exec.c:4143 fe-exec.c:4274
#, c-format
msgid "invalid multibyte character"
msgstr "不正なマルチバイト文字"

#: fe-gssapi-common.c:122
msgid "GSSAPI name import error"
msgstr "GSSAPI名のインポートエラー"

#: fe-lobj.c:144 fe-lobj.c:207 fe-lobj.c:397 fe-lobj.c:487 fe-lobj.c:560
#: fe-lobj.c:956 fe-lobj.c:963 fe-lobj.c:970 fe-lobj.c:977 fe-lobj.c:984
#: fe-lobj.c:991 fe-lobj.c:998 fe-lobj.c:1005
#, c-format
msgid "cannot determine OID of function %s"
msgstr "関数%sのOIDが特定できません"

#: fe-lobj.c:160
#, c-format
msgid "argument of lo_truncate exceeds integer range"
msgstr "lo_truncateの引数が整数の範囲を超えています"

#: fe-lobj.c:262
#, c-format
msgid "argument of lo_read exceeds integer range"
msgstr "lo_readの引数が整数の範囲を超えています"

#: fe-lobj.c:313
#, c-format
msgid "argument of lo_write exceeds integer range"
msgstr "lo_writeの引数が整数範囲を超えています"

#: fe-lobj.c:669 fe-lobj.c:780
#, c-format
msgid "could not open file \"%s\": %s"
msgstr "ファイル\"%s\"をオープンできませんでした: %s"

#: fe-lobj.c:725
#, c-format
msgid "could not read from file \"%s\": %s"
msgstr "ファイル\"%s\"を読み込めませんでした: %s"

#: fe-lobj.c:801 fe-lobj.c:824
#, c-format
msgid "could not write to file \"%s\": %s"
msgstr "ファイル\"%s\"に書き込めませんでした: %s"

#: fe-lobj.c:908
#, c-format
msgid "query to initialize large object functions did not return data"
msgstr "ラージオブジェクト機能を初期化する問い合わせがデータを返しませんでした"

#: fe-misc.c:239
#, c-format
msgid "integer of size %lu not supported by pqGetInt"
msgstr "サイズ%luの整数はpqGetIntでサポートされていません"

#: fe-misc.c:272
#, c-format
msgid "integer of size %lu not supported by pqPutInt"
msgstr "サイズ%luの整数はpqPutIntでサポートされていません"

#: fe-misc.c:791 fe-secure-openssl.c:181 fe-secure-openssl.c:287
#: fe-secure.c:222 fe-secure.c:389
#, c-format
msgid ""
"server closed the connection unexpectedly\n"
"\tThis probably means the server terminated abnormally\n"
"\tbefore or while processing the request."
msgstr ""
"サーバーとの接続が想定外にクローズされました\n"
"        おそらく要求の処理前または処理中にサーバーが異常終了\n"
"        したことを意味しています。"

#: fe-misc.c:858
msgid "connection not open\n"
msgstr "接続はオープンされていません\n"

#: fe-misc.c:1046
#, c-format
msgid "timeout expired"
msgstr "タイムアウト期間が過ぎました"

#: fe-misc.c:1098
#, c-format
msgid "invalid socket"
msgstr "不正なソケットです"

#: fe-misc.c:1121
#, c-format
msgid "%s() failed: %s"
msgstr "%s() が失敗しました: %s"

#: fe-protocol3.c:188
#, c-format
msgid "message type 0x%02x arrived from server while idle"
msgstr "待機中にサーバーからメッセージ種類0x%02xが届きました"

#: fe-protocol3.c:401
#, c-format
msgid "server sent data (\"D\" message) without prior row description (\"T\" message)"
msgstr "サーバーが先行の行記述(\"T\"メッセージ)なしでデータ(\"D\"メッセージ)を送信しました"

#: fe-protocol3.c:443
#, c-format
msgid "unexpected response from server; first received character was \"%c\""
msgstr "サーバーから想定外の応答がありました。受け付けた先頭文字は\"%c\"です"

#: fe-protocol3.c:462
#, c-format
msgid "message contents do not agree with length in message type \"%c\""
msgstr "メッセージの内容がメッセージタイプ\"%c\"での長さと合っていません"

#: fe-protocol3.c:480
#, c-format
msgid "lost synchronization with server: got message type \"%c\", length %d"
msgstr "サーバーとの同期が失われました。受信したメッセージタイプは\"%c\"、長さは%d"

#: fe-protocol3.c:532 fe-protocol3.c:572
msgid "insufficient data in \"T\" message"
msgstr "\"T\"メッセージ内のデータが不十分です"

#: fe-protocol3.c:643 fe-protocol3.c:849
msgid "out of memory for query result"
msgstr "問い合わせ結果用のメモリが不足しています"

#: fe-protocol3.c:712
msgid "insufficient data in \"t\" message"
msgstr "\"t\"メッセージ内のデータが足りません"

#: fe-protocol3.c:771 fe-protocol3.c:803 fe-protocol3.c:821
msgid "insufficient data in \"D\" message"
msgstr "\"D\"\"メッセージ内のデータが不十分です"

#: fe-protocol3.c:777
msgid "unexpected field count in \"D\" message"
msgstr "\"D\"メッセージ内のフィールド数が想定外です。"

#: fe-protocol3.c:1033
msgid "no error message available\n"
msgstr "エラーメッセージがありません\n"

#. translator: %s represents a digit string
#: fe-protocol3.c:1081 fe-protocol3.c:1100
#, c-format
msgid " at character %s"
msgstr "(文字位置: %s)"

#: fe-protocol3.c:1113
#, c-format
msgid "DETAIL:  %s\n"
msgstr "DETAIL:  %s\n"

#: fe-protocol3.c:1116
#, c-format
msgid "HINT:  %s\n"
msgstr "HINT:  %s\n"

#: fe-protocol3.c:1119
#, c-format
msgid "QUERY:  %s\n"
msgstr "QUERY:  %s\n"

#: fe-protocol3.c:1126
#, c-format
msgid "CONTEXT:  %s\n"
msgstr "CONTEXT:  %s\n"

#: fe-protocol3.c:1135
#, c-format
msgid "SCHEMA NAME:  %s\n"
msgstr "SCHEMA NAME:  %s\n"

#: fe-protocol3.c:1139
#, c-format
msgid "TABLE NAME:  %s\n"
msgstr "TABLE NAME:  %s\n"

#: fe-protocol3.c:1143
#, c-format
msgid "COLUMN NAME:  %s\n"
msgstr "COLUMN NAME:  %s\n"

#: fe-protocol3.c:1147
#, c-format
msgid "DATATYPE NAME:  %s\n"
msgstr "DATATYPE NAME:  %s\n"

#: fe-protocol3.c:1151
#, c-format
msgid "CONSTRAINT NAME:  %s\n"
msgstr "CONSTRAINT NAME:  %s\n"

#: fe-protocol3.c:1163
msgid "LOCATION:  "
msgstr "LOCATION:  "

#: fe-protocol3.c:1165
#, c-format
msgid "%s, "
msgstr "%s, "

#: fe-protocol3.c:1167
#, c-format
msgid "%s:%s"
msgstr "%s:%s"

#: fe-protocol3.c:1362
#, c-format
msgid "LINE %d: "
msgstr "行 %d: "

#: fe-protocol3.c:1424
#, c-format
msgid "received invalid protocol negotiation message: server requested downgrade to a higher-numbered version"
msgstr "不正なプロトコルネゴシエーションメッセージを受信: サーバーがより大きなバージョン番号へのダウングレードを要求しました"

#: fe-protocol3.c:1430
#, c-format
msgid "received invalid protocol negotiation message: server requested downgrade to pre-3.0 protocol version"
msgstr "不正なプロトコルネゴシエーションメッセージを受信: サーバーが3.0より前のプロトコルバージョンへのダウングレードを要求しました"

#: fe-protocol3.c:1437
#, c-format
msgid "received invalid protocol negotiation message: server requested downgrade to non-existent 3.1 protocol version"
msgstr "不正なプロトコルネゴシエーションメッセージを受信: サーバーが、存在しない 3.1 プロトコルバージョンへのダウングレードを要求しました"

#: fe-protocol3.c:1443
#, c-format
msgid "received invalid protocol negotiation message: server reported negative number of unsupported parameters"
msgstr "不正なプロトコルネゴシエーションメッセージを受信: サーバーがサポート外パラメータの数として負数を通知しました"

#: fe-protocol3.c:1449
#, c-format
msgid "received invalid protocol negotiation message: server negotiated but asks for no changes"
msgstr "不正なプロトコルネゴシエーションメッセージを受信: サーバーがネゴシエーションを行いましたが、変更要求がありませんでした"

#: fe-protocol3.c:1455
#, c-format
msgid "server only supports protocol version %d.%d, but \"%s\" was set to %d.%d"
msgstr "サーバーはプロトコルバージョン %d.%d のみサポートしていますが、\"%s\"は %d.%d に設定されています"

#: fe-protocol3.c:1480
#, c-format
msgid "received invalid protocol negotiation message: server reported unsupported parameter name without a \"%s\" prefix (\"%s\")"
msgstr "不正なプロトコルネゴシエーションメッセージを受信: サーバーが\"%s\"プレフィクスのないサポートされていないパラメータ名(\"%s\")を報告しました"

#: fe-protocol3.c:1483
#, c-format
msgid "received invalid protocol negotiation message: server reported an unsupported parameter that was not requested (\"%s\")"
msgstr "不正なプロトコルネゴシエーションメッセージを受信: サーバーが要求されていないサポート外のパラメータ(\"%s\")を通知しました"

#: fe-protocol3.c:1490
#, c-format
msgid "received invalid protocol negotiation message: message too short"
msgstr "不正なプロトコルネゴシエーションメッセージを受信: メッセージが短すぎます"

#: fe-protocol3.c:1894
#, c-format
msgid "PQgetline: not doing text COPY OUT"
msgstr "PQgetline: テキストのCOPY OUTを行っていません"

#: fe-protocol3.c:2269
#, c-format
msgid "protocol error: no function result"
msgstr "プロトコルエラー: 関数の結果がありません"

#: fe-protocol3.c:2281
#, c-format
msgid "protocol error: id=0x%x"
msgstr "プロトコルエラー: id=0x%x"

#: fe-secure-common.c:123
#, c-format
msgid "SSL certificate's name contains embedded null"
msgstr "SSL証明書の名前の途中にnullが含まれています"

#: fe-secure-common.c:228
#, c-format
msgid "certificate contains IP address with invalid length %zu"
msgstr "証明書は不正な長さ%zuのIPアドレスを含んでいます"

#: fe-secure-common.c:237
#, c-format
msgid "could not convert certificate's IP address to string: %s"
msgstr "証明書のIPアドレスを文字列へ変換できませんでした: %s"

#: fe-secure-common.c:269
#, c-format
msgid "host name must be specified for a verified SSL connection"
msgstr "検証SSL接続ではホスト名を指定しなければなりません"

#: fe-secure-common.c:286
#, c-format
msgid "server certificate for \"%s\" (and %d other name) does not match host name \"%s\""
msgid_plural "server certificate for \"%s\" (and %d other names) does not match host name \"%s\""
msgstr[0] "\"%s\"のサーバー証明書(および%d個のほかの名前)はホスト名\"%s\"と一致しません"

#: fe-secure-common.c:294
#, c-format
msgid "server certificate for \"%s\" does not match host name \"%s\""
msgstr "\"%s\"のサーバー証明書がホスト名\"%s\"とマッチしません"

#: fe-secure-common.c:299
#, c-format
msgid "could not get server's host name from server certificate"
msgstr "サーバー証明書からサーバーのホスト名を取得できませんでした"

#: fe-secure-gssapi.c:201
msgid "GSSAPI wrap error"
msgstr "GSSAPI名ラップエラー"

#: fe-secure-gssapi.c:208
#, c-format
msgid "outgoing GSSAPI message would not use confidentiality"
msgstr "送出されるGSSAPIメッセージに機密性が適用されません"

#: fe-secure-gssapi.c:215 fe-secure-gssapi.c:723
#, c-format
msgid "client tried to send oversize GSSAPI packet (%zu > %zu)"
msgstr "クライアントは過大なGSSAPIパケットを送信しようとしました: (%zu > %zu)"

#: fe-secure-gssapi.c:354 fe-secure-gssapi.c:599
#, c-format
msgid "oversize GSSAPI packet sent by the server (%zu > %zu)"
msgstr "過大なGSSAPIパケットがサーバーから送出されました: (%zu > %zu)"

#: fe-secure-gssapi.c:393
msgid "GSSAPI unwrap error"
msgstr "GSSAPIアンラップエラー"

#: fe-secure-gssapi.c:402
#, c-format
msgid "incoming GSSAPI message did not use confidentiality"
msgstr "到着したGSSAPIメッセージには機密性が適用されていません"

#: fe-secure-gssapi.c:662
msgid "could not initiate GSSAPI security context"
msgstr "GSSAPIセキュリティコンテキストを開始できませんでした"

#: fe-secure-gssapi.c:712
msgid "GSSAPI size check error"
msgstr "GSSAPIサイズチェックエラー"

#: fe-secure-openssl.c:185 fe-secure-openssl.c:291 fe-secure-openssl.c:1378
#, c-format
msgid "SSL SYSCALL error: %s"
msgstr "SSL SYSCALLエラー: %s"

#: fe-secure-openssl.c:191 fe-secure-openssl.c:297 fe-secure-openssl.c:1381
#, c-format
msgid "SSL SYSCALL error: EOF detected"
msgstr "SSL SYSCALLエラー: EOFを検出"

#: fe-secure-openssl.c:201 fe-secure-openssl.c:307 fe-secure-openssl.c:1389
#, c-format
msgid "SSL error: %s"
msgstr "SSLエラー: %s"

#: fe-secure-openssl.c:215 fe-secure-openssl.c:321
#, c-format
msgid "SSL connection has been closed unexpectedly"
msgstr "SSL接続が意図せずにクローズされました"

#: fe-secure-openssl.c:220 fe-secure-openssl.c:326 fe-secure-openssl.c:1436
#, c-format
msgid "unrecognized SSL error code: %d"
msgstr "認識できないSSLエラーコード: %d"

#: fe-secure-openssl.c:368
#, c-format
msgid "could not determine server certificate signature algorithm"
msgstr "サーバー証明書の署名アルゴリズムを特定できませんでした"

#: fe-secure-openssl.c:388
#, c-format
msgid "could not find digest for NID %s"
msgstr "NID %sのダイジェストが見つかりませんでした"

#: fe-secure-openssl.c:397
#, c-format
msgid "could not generate peer certificate hash"
msgstr "接続先の証明書ハッシュの生成に失敗しました"

#: fe-secure-openssl.c:479
#, c-format
msgid "SSL certificate's name entry is missing"
msgstr "SSL証明書に名前のエントリがありません"

#: fe-secure-openssl.c:509
#, c-format
msgid "SSL certificate's address entry is missing"
msgstr "SSL証明書のアドレスのエントリがありません"

#: fe-secure-openssl.c:714
#, c-format
msgid "could not open SSL key logging file \"%s\": %s"
msgstr "SSLキー記録ファイル\"%s\"をオープンできませんでした: %s"

#: fe-secure-openssl.c:722
#, c-format
msgid "could not write to SSL key logging file \"%s\": %s"
msgstr "SSLキー記録ファイル\"%s\"に書き込めませんでした: %s"

#: fe-secure-openssl.c:775
#, c-format
msgid "could not create SSL context: %s"
msgstr "SSLコンテキストを作成できませんでした: %s"

#: fe-secure-openssl.c:817
#, c-format
msgid "invalid value \"%s\" for minimum SSL protocol version"
msgstr "SSLプロトコル最小バージョンに対する不正な値\"%s\""

#: fe-secure-openssl.c:827
#, c-format
msgid "could not set minimum SSL protocol version: %s"
msgstr "SSLプロトコル最小バージョンを設定できませんでした: %s"

#: fe-secure-openssl.c:843
#, c-format
msgid "invalid value \"%s\" for maximum SSL protocol version"
msgstr "SSLプロトコル最大バージョンに対する不正な値\"%s\""

#: fe-secure-openssl.c:853
#, c-format
msgid "could not set maximum SSL protocol version: %s"
msgstr "SSLプロトコル最大バージョンを設定できませんでした: %s"

#: fe-secure-openssl.c:891
#, c-format
msgid "could not load system root certificate paths: %s"
msgstr "システムルート証明書パスをロードできませんでした: %s"

#: fe-secure-openssl.c:908
#, c-format
msgid "could not read root certificate file \"%s\": %s"
msgstr "ルート証明書ファイル\"%s\"を読み取れませんでした: %s"

#: fe-secure-openssl.c:960
#, c-format
msgid ""
"could not get home directory to locate root certificate file\n"
"Either provide the file, use the system's trusted roots with sslrootcert=system, or change sslmode to disable server certificate verification."
msgstr ""
"ルート証明書ファイルを特定するためのホームディレクトリが取得できませんでした\n"
"ファイルを用意する、 sslrootcert=systemでシステムの信頼済みルート証明書を使用する、または sslmode を変更してサーバー証明書の検証を無効にしてください。"

#: fe-secure-openssl.c:963
#, c-format
msgid ""
"root certificate file \"%s\" does not exist\n"
"Either provide the file, use the system's trusted roots with sslrootcert=system, or change sslmode to disable server certificate verification."
msgstr ""
"ルート証明書ファイル\"%s\"が存在しません\n"
"ファイルを用意する、sslrootcert=systemでシステムの信頼済みルート証明書を使用する、またはsslmodeを変更してサーバー証明書の検証を無効にしてください。"

#: fe-secure-openssl.c:998
#, c-format
msgid "could not open certificate file \"%s\": %s"
msgstr "証明書ファイル\"%s\"をオープンできませんでした: %s"

#: fe-secure-openssl.c:1016
#, c-format
msgid "could not read certificate file \"%s\": %s"
msgstr "証明書ファイル\"%s\"を読み込めませんでした: %s"

#: fe-secure-openssl.c:1040
#, c-format
msgid "could not establish SSL connection: %s"
msgstr "SSL接続を確立できませんでした: %s"

#: fe-secure-openssl.c:1053
#, c-format
msgid "WARNING: sslkeylogfile support requires OpenSSL\n"
msgstr "警告: sslkeylogfile をサポートするにはOpenSSLが必要です\n"

#: fe-secure-openssl.c:1055
#, c-format
msgid "WARNING: libpq was not built with sslkeylogfile support\n"
msgstr "警告: libpq は sslkeylogfile のサポート無しでビルドされています\n"

#: fe-secure-openssl.c:1086
#, c-format
msgid "could not set SSL Server Name Indication (SNI): %s"
msgstr "SSLサーバー名表示(SNI)を設定できませんでした: %s"

#: fe-secure-openssl.c:1103
#, c-format
msgid "could not set SSL ALPN extension: %s"
msgstr "SSL ALPN拡張を設定できませんでした: %s"

#: fe-secure-openssl.c:1146
#, c-format
msgid "could not load SSL engine \"%s\": %s"
msgstr "SSLエンジン\"%s\"を読み込みできませんでした: %s"

#: fe-secure-openssl.c:1157
#, c-format
msgid "could not initialize SSL engine \"%s\": %s"
msgstr "SSLエンジン\"%s\"を初期化できませんでした: %s"

#: fe-secure-openssl.c:1172
#, c-format
msgid "could not read private SSL key \"%s\" from engine \"%s\": %s"
msgstr "SSL秘密鍵\"%s\"をエンジン\"%s\"から読み取れませんでした: %s"

#: fe-secure-openssl.c:1185
#, c-format
msgid "could not load private SSL key \"%s\" from engine \"%s\": %s"
msgstr "SSL秘密鍵\"%s\"をエンジン\"%s\"から読み取れませんでした: %s"

#: fe-secure-openssl.c:1222
#, c-format
msgid "certificate present, but not private key file \"%s\""
msgstr "証明書はありますが、秘密鍵ファイル\"%s\"はありません"

#: fe-secure-openssl.c:1225
#, c-format
msgid "could not stat private key file \"%s\": %m"
msgstr "秘密鍵ファイル\"%s\"をstatできませんでした: %m"

#: fe-secure-openssl.c:1233
#, c-format
msgid "private key file \"%s\" is not a regular file"
msgstr "秘密鍵ファイル\"%s\"は通常のファイルではありません"

#: fe-secure-openssl.c:1266
#, c-format
msgid "private key file \"%s\" has group or world access; file must have permissions u=rw (0600) or less if owned by the current user, or permissions u=rw,g=r (0640) or less if owned by root"
msgstr "秘密鍵ファイル\"%s\"はグループに対して、もしくは無制限にアクセスを許可しています; ファイルのパーミッションは u=rw (0600) かそれよりも狭い必要があります、rootが所有している場合は  u=rw,g=r (0640) かそれよりも狭い必要があります"

#: fe-secure-openssl.c:1290
#, c-format
msgid "could not load private key file \"%s\": %s"
msgstr "秘密鍵ファイル\"%s\"をロードできませんでした: %s"

#: fe-secure-openssl.c:1306
#, c-format
msgid "certificate does not match private key file \"%s\": %s"
msgstr "証明書と秘密鍵ファイル\"%s\"が一致しません: %s"

#: fe-secure-openssl.c:1375
#, c-format
msgid "SSL error: certificate verify failed: %s"
msgstr "SSLエラー: 証明書の検証に失敗しました: %s"

#: fe-secure-openssl.c:1420
#, c-format
msgid "This may indicate that the server does not support any SSL protocol version between %s and %s."
msgstr "このことは、クライアントがSSLプロトコルのバージョン%sから%sの間のいずれもサポートしていないことを示唆しているかもしれません。"

#: fe-secure-openssl.c:1452
#, c-format
msgid "direct SSL connection was established without ALPN protocol negotiation extension"
msgstr "直接SSL接続がALPNプロトコルネゴシエーション拡張なしで確立されました"

#: fe-secure-openssl.c:1464
#, c-format
msgid "SSL connection was established with unexpected ALPN protocol"
msgstr "SSL接続が想定外のALPNプロトコルで確立されました"

#: fe-secure-openssl.c:1481
#, c-format
msgid "certificate could not be obtained: %s"
msgstr "証明書を取得できませんでした: %s"

#: fe-secure-openssl.c:1560
#, c-format
msgid "no SSL error reported"
msgstr "SSLエラーはありませんでした"

#: fe-secure-openssl.c:1603
#, c-format
msgid "SSL error code %lu"
msgstr "SSLエラーコード: %lu"

#: fe-secure-openssl.c:1905
#, c-format
msgid "WARNING: sslpassword truncated\n"
msgstr "警告: sslpasswordが切り詰められました\n"

#: fe-secure.c:233
#, c-format
msgid "could not receive data from server: %s"
msgstr "サーバーからデータを受信できませんでした: %s"

#: fe-secure.c:404
#, c-format
msgid "could not send data to server: %s"
msgstr "サーバーにデータを送信できませんでした: %s"

#: win32.c:310
#, c-format
msgid "unrecognized socket error: 0x%08X/%d"
msgstr "不明なソケットエラー 0x%08X/%d"
